src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsGetController.ts

```diff
@@ -38,8 +38,16 @@ export default class ConfirmContactDetailsGetController extends GetController {
     req.session.userCase.applicant1Address1 = 'Flat 100';
     req.session.userCase.applicant1Address2 = 'Plashet Grove';
     req.session.userCase.applicant1AddressTown = 'London';
-    req.session.userCase.applicant1PhoneNumber = '1234567890';
-    req.session.userCase.applicant1EmailAddress = 'test@gmail.com';
+    req.session.userCase.applicant1PhoneNumber = '';
```

Comment:
why we initialising this field to blank? can this field be always intended to be blank?